src/CattyTests/Bricks/GlideToBrickTests.swift

```diff
@@ -37,4 +37,40 @@ final class GlideToBrickTests: XCTestCase {
         XCTAssertEqual(brick.xDestination, brick.formula(forLineNumber: 1, andParameterNumber: 0))
         XCTAssertEqual(brick.yDestination, brick.formula(forLineNumber: 1, andParameterNumber: 1))
     }
+
+    func testMutableCopy() {
+        let brick = GlideToBrick()
+
+        brick.durationInSeconds = Formula(double: 1)
+        brick.xDestination = Formula(double: 1)
+        brick.yDestination = Formula(double: 1)
+
+        let copiedBrick: GlideToBrick = brick.mutableCopy(with: CBMutableCopyContext(), andErrorReporting: true) as! GlideToBrick
```

Comment:
Since we want to make absolutely sure that the copied Brick is not only equal to the original Brick, but also references a separate instance, I would suggest adding one more assert for each copiedBrick, e.g. something like _XCTAssertFalse(brick === copiedBrick)_.